src/CoreWCF.Primitives/src/CoreWCF/Channels/IWebMessageEncoderHelper.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace CoreWCF.Channels
+{
+    public interface IWebMessageEncoderHelper
+    {
```

Comment:
I don't think this is needed as I believe it's for supporting the 4.0 compat behavior with the fake XmlReader. We should just have all code expect the latest behavior.

Reply:
This was addressed in b0c21a3804140f47b064720903835a1427a24357.